### PR TITLE
Add script for removing Inductor dependencies from Inductor generated code

### PR DESCRIPTION
**Pull request by Chillee**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #123335
* __->__ #125811

Usage:
```python
TORCHINDUCTOR_UNIQUE_KERNEL_NAMES=1 python foo.py
TORCHINDUCTOR_DUMP_LAUNCH_PARAMS=1 python /tmp/torchinductor_chilli/js/cjsbczkf6fj36nhaxxypll6cy4fmwmkoauklrgrvuody2mn7oeef.py
python torch/utils/_get_clean_triton.py /tmp/torchinductor_chilli/js/cjsbczkf6fj36nhaxxypll6cy4fmwmkoauklrgrvuody2mn7oeef.py
```

Example generated code: https://pastebin.com/m6Ae8heB

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

### CODE REVIEW

**Review by chenyang78 — APPROVED** [review]
LGTM. Thanks!

File: test/inductor/test_kernel_benchmark.py [hunk 1]
```diff
@@ -59,6 +60,39 @@ def verify_compiled_kernels(self, GB_count=1):
...
 
+    def verify_remove_inductor_deps(self, compiled_module):
+        try:
+            out = subprocess.check_output(
+                f"{sys.executable} {compiled_module.__file__}".split(),
+                env={**os.environ.copy(), "TORCHINDUCTOR_DUMP_LAUNCH_PARAMS": "1"},
+                stderr=subprocess.STDOUT,
+            )
+        except subprocess.CalledProcessError as e:
+            breakpoint()
```
Comment (drisspg) [inline test/inductor/test_kernel_benchmark.py:71]: 🫥

Comment (Chillee) [reply test/inductor/test_kernel_benchmark.py:71]: oh lol

**Comment by Chillee** [comment]
failures look unrelated?

**Pull request closed without merging**